package.json

```diff
@@ -75,6 +87,7 @@
     "eslint-plugin-prettier": "^2.2.0",
     "eslint-plugin-react": "^7.7.0",
     "file-loader": "^1.1.5",
+    "filereader-stream": "^2.0.0",
```

Comment:
I'm hopeful that this and `stream-http` can become dependencies of `music-metadata-browser` and that Webamp won't need to know about them.

Reply:
I will have a look to directly support parsing from a browser File object.

I am not sure if that is a good idea to embed an http client. HTTP is world on it's own, there are many different http clients. If security for example is an important aspect, things can become tricky.

At the other hand, stream is a great abstraction, but it does not offer the ability to  jump (read seek or fast forward in the data).